.gitignore

```diff
@@ -1,4 +1,5 @@
-/attachments/
+/attachments/*
+!/attachments/.gitkeep
```

Comment:
Oh, interesting. I just merged it into develop. Do you want me to touch it up, or should we just keep it as is?